sambacc/ctdb.py

```diff
@@ -298,7 +312,9 @@ def _node_check(pnn: int, nodes_json: str, real_path: str) -> bool:
     return True
 
 
-def _node_update_check(json_data, nodes_json: str, real_path: str):
+def _node_update_check(
+    json_data: dict[str, typing.Any], nodes_json: str, real_path: str
```

Comment:
`nodes_json` is not used in this function. With type hinting, functions signatures become longer; maybe it would be better to avoid used arguments.

Reply:
Function is private so it should be straightforward.